addons/account/models/account_bank_statement.py

```diff
@@ -96,8 +96,8 @@ class AccountBankStatement(models.Model):
         compute='_compute_problem_description',
     )
 
-    attachment_ids = fields.Many2many(
-        comodel_name='ir.attachment',
+    attachment_ids = fields.One2many(
+        related='linked_attachment_ids',
```

Comment:
Why don't you simply remove `attachment_ids` and use `linked_attachment_ids` instead?

Reply:
Good point, especially because of the `compute_sudo=True` for related fields (wtf btw :zany_face: )